projects/gateway2/deployer/values.go

```diff
@@ -85,10 +85,11 @@ type helmImage struct {
 }
 
 type helmService struct {
-	Type             *string           `json:"type,omitempty"`
-	ClusterIP        *string           `json:"clusterIP,omitempty"`
-	ExtraAnnotations map[string]string `json:"extraAnnotations,omitempty"`
-	ExtraLabels      map[string]string `json:"extraLabels,omitempty"`
+	Type                  *string           `json:"type,omitempty"`
```

Comment:
Why does `helmService` redefine the `Service` type from `projects/gateway2/api/v1alpha1/kube_types.go` instead of just wrapping it?

Reply:
this may have been added before we had kube_types (before, the GWP API was defined in protos) and also there was initially some pushback on exposing every value via helm. but i agree we should just import the whole service struct from kube_types. (but i think it doesn't need to be done in this PR)